### ISSUE TITLE
feature request: Nightshade and elderberries compost

### ISSUE THREAD

**Issue by mkire** [Lothrazar/RootsClassic#97]
would you please tag nightshade and elderberry for the composter, in addition to the currants?

**Comment by Lothrazar** [comment]
yes great idea. let them go in composter. is doing it in mc 1.20.1 only enough or do you play older versions

**Comment by mkire** [comment]
1.16.5, 1.18.2, and 1.19.2 are the versions i'm playing, depending on the modpack.

honestly the request was just for whatever versions are still getting updated.

**Comment by Mrbysco** [comment]
In both 1.20.1 and 1.21.1 they should be compostable
